admin/tests/unit/services/access-control-test.js

```diff
@@ -196,9 +196,9 @@ module('Unit | Service | access-control', function (hooks) {
         { role: 'isSuperAdmin', hasAccess: true },
         { role: 'isSupport', hasAccess: true },
         { role: 'isMetier', hasAccess: true },
-        { role: 'isCertif', hasAccess: false },
+        { role: 'isCertif', hasAccess: true },
       ].forEach(function ({ role, hasAccess }) {
-        test(`should be ${hasAccess} if current admin member is ${role}`, function (assert) {
+        test(`should be accessible for all role members`, function (assert) {
```

Comment:
Bien que ce soit vrai, est-ce qu'on veut 4 tests successifs avec le même nom ?
```suggestion
        test(`should be accessible for all role members (${role} access)`, function (assert) {
```